examples/resizable/index.html

```diff
@@ -0,0 +1,23 @@
+<html>
+    <head>
+        <title>Jabberwock Editor</title>
+        <link rel="shortcut icon" href="#" />
+    </head>
+    <body style="font-family: sans-serif">
+        <menu style="background-color: #999; height:120px; text-align: center; font-size: 24px; color: white">menu</menu>
+        <main>
+            <h1 style="text-align: center;">In page title.</h1>
+            <p style="text-align: center;">Editor : </p>
+            <div style="border: 1px solid #bbb; margin: 10px 15%;">
```

Comment:
I would not customize so much the demo page as this only add "noise" and not really add "value".
Unless it has some interesting style regarding the Jabberwock story.